packages/clerk-js/src/core/clerk.test.ts

```diff
@@ -1458,13 +1458,13 @@ describe('Clerk singleton', () => {
   });
 
   describe('buildUrlWithAuth', () => {
-    it('builds an absolute url from a relative url in development for url based session syncing', async () => {
+    it('builds a relative url from a relative url in development for url based session syncing', async () => {
```

Comment:
❓ Should we add a new test for the `absolute url` case ?

Reply:
It's not supposed to build an absolute url in that case anymore.